src/data/repositories/utils/AlertOutbreakMapper.ts

```diff
@@ -0,0 +1,72 @@
+import { D2TrackerTrackedEntity } from "@eyeseetea/d2-api/api/trackerTrackedEntities";
+import {
+    DataSource,
+    IncidentStatusType,
+} from "../../../domain/entities/disease-outbreak-event/DiseaseOutbreakEvent";
+import { AlertOptions } from "../../../domain/repositories/AlertRepository";
+import { Maybe } from "../../../utils/ts-utils";
+import { Option } from "../../../domain/entities/Ref";
+import { NotificationOptions } from "../../../domain/repositories/NotificationRepository";
+import { alertOutbreakCodes } from "../consts/DiseaseOutbreakConstants";
+
+export function mapTrackedEntityAttributesToAlertOutbreak(
+    nationalTrackedEntity: D2TrackerTrackedEntity,
+    alertTrackedEntity: D2TrackerTrackedEntity
+): AlertOptions {
+    if (!nationalTrackedEntity.trackedEntity) throw new Error("Tracked entity not found");
+
+    const diseaseOutbreak: AlertOptions = {
+        eventId: nationalTrackedEntity.trackedEntity,
+        dataSource: getValueFromMap("dataSource", nationalTrackedEntity) as DataSource,
```

Comment:
Also when you bring the development changes you will see in DiseaseOutbreakMapper that we try to avoid the use of `as` making the code more type safe